Docs/source/conf.py

```diff
@@ -131,14 +138,16 @@ def __init__(self, *args, **kwargs):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = "sphinx_rtd_theme"
 
 numfig = True
 math_eqref_format = "{number}"
-numfig_format = {'figure': 'Fig. %s',
```

Comment:
I like this formatting better. Is there an option with ruff to use this style?

Reply:
Nope, it just uses [black style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html). The style is intentional, the extra break reduces the amount of lines changed when adding/removing new arguments in PRs to dictionaries.